xbmc/platform/linux/PlatformWebOS.cpp

```diff
@@ -28,7 +28,12 @@ bool CPlatformWebOS::InitStageOne()
   setenv("XKB_CONFIG_ROOT", "/usr/share/X11/xkb", 1);
   setenv("WAYLAND_DISPLAY", "wayland-0", 1);
   setenv("PYTHONHOME", (HOME + "/lib/python3").c_str(), 1);
-  setenv("PYTHONPATH", (HOME + "/lib/python3").c_str(), 1);
+
+  std::string pythonPath;
+  pythonPath = (HOME + "/lib/python3").c_str();
+  pythonPath += (":" + pythonPath + "/site-packages").c_str();
```

Comment:
i wouldnt expect you need the c_str() in either of the above two lines? Its going into a string, not a char*